tests/core/contracts/test_contract_method_to_argument_matching.py

```diff
@@ -168,22 +169,69 @@ def test_finds_function_with_matching_args_non_strict(
 
 def test_finds_function_with_matching_args_strict_type_checking_by_default(w3):
     contract = w3.eth.contract(abi=MULTIPLE_FUNCTIONS)
-    with pytest.raises(MismatchedABI):
+    with pytest.raises(
+        MismatchedABI,
+        match=re.escape(
+            "\nABI Not Found!\n"
+            "Multiple elements were found matching 1 arguments.\n"
+            "Provided argument types: (str)\n"
+            "Provided keyword argument types: {}\n\n"
+            "Encountered problems with the following elements named `a`.\n"
+            "a()\n"
+            "Function `a` expects 0 arguments but 1 were given.\n"
+            "a(bytes32)\n"
+            "Error! Could not encode argument 1 value `` as `bytes32`.\n"
+            "a(uint256)\n"
+            "Error! Could not encode argument 1 value `` as `uint256`.\n"
+            "a(uint8)\n"
+            "Error! Could not encode argument 1 value `` as `uint8`.\n"
+            "a(int8)\n"
+            "Error! Could not encode argument 1 value `` as `int8`.\n"
+            "a((int256,bool)[])\n"
+            "Error! Arguments do not match types in `a((int256,bool)[])`.\n"
+        ),
+    ):
         contract._find_matching_fn_abi("a", *[""])
 
 
 def test_error_when_duplicate_match(w3):
     Contract = w3.eth.contract(abi=MULTIPLE_FUNCTIONS)
 
-    with pytest.raises(MismatchedABI):
+    with pytest.raises(
+        MismatchedABI,
+        match=re.escape(
+            "\nABI Not Found!\n"
+            "Multiple elements were found matching 1 arguments.\n"
```

Comment:
I'm not sure if you can make this wording work throughout, but maybe something like: `Multiple elements were found that accept 1 arguments`

Reply:
I think it may be as simple as adding `(s)`. I made a comment in the actual messages under the abi utils. Do you think that clarifies well enough?

Reply:
The word `matching` is what's confusing to me. To me, this says the element matches the argument (which I know is not the intent) but changing to to `accepts` clears it up. Either way, feel free to take or leave.